modules/nf-core/epang/split/main.nf

```diff
@@ -0,0 +1,37 @@
+process EPANG_SPLIT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::epa-ng=0.3.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/epa-ng:0.3.8--h9a82719_1':
+        'quay.io/biocontainers/epa-ng:0.3.8--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(refaln)
+    path  fullaln
```

Comment:
Consider if this should be part of the tuple in the other input channel? I'm not sure.

Reply:
It could well be since it's a required parameter. Thanks for the suggestion.